zipline/lib/adjust.pyd

```diff
@@ -0,0 +1,607 @@
+cdef class Adjustment:
```

Comment:
pxd files are like headers so that the functions can be cimported in other cython files. you need the function signatures and class member definitions but that's it.